src/web/components/SlotBodyEnd.tsx

```diff
@@ -32,7 +32,7 @@ export const SlotBodyEnd = ({
             campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
             abTestName: 'remote_epic_test',
             abTestVariant: 'api',
-            referrerUrl: window?.location.href.split('?')[0],
+            referrerUrl: window?.location.origin + window.location.pathname
```

Comment:
@oliverlloyd I thought we didn't support optional chaining - `window?`?

Reply:
I thought it add been added 🤔It does work tho so it must have been!

Reply:
Yeh, I think we had an issue with not supporting in Babel even though we were on the right TS so maybe this doesn't go through that compile step or something something

Reply:
I keep trying to use it and getting snags with tooling. Might be best to hold off.

Reply:
I can get rid of it but it's not new to this PR. We're already using it in in this component at least, in production 🤔

Reply:
Possibly not relevant if we decide we can't use optional chaining, but if we do want to use it then presumably we need `window?.location.pathname` in the second case too?

Reply:
Btw, optional chaining is only added here so the component fails more gracefully if it's rendered on the server for whatever reason.

Reply:
From memory, I think I hit issues with Jest so it could be if you're not using that you might be fine. Also, now that we're all aware of the potential for tooling issues it won't be so hard to debug if it pops up!

Reply:
I see. Not using Jest to test this here so we should be fine. Going to merge as is and keep an eye on it once it's on prod. Pretty sure we'll be fine 🤞